src/mon/ConfigMap.cc

```diff
@@ -168,3 +168,42 @@ bool ConfigMap::parse_mask(
   }
   return true;
 }
+
+
+// --------------
+
+void ConfigChangeSet::dump(Formatter *f) const
+{
+  f->dump_int("version", version);
+  f->dump_stream("timestamp") << stamp;
+  f->dump_string("name", name);
+  f->open_array_section("changes");
+  for (auto& i : diff) {
+    f->open_object_section("change");
```

Comment:
This should probably be outputting the key `i.first` as well as the values

Reply:
fixed!